api/src/pcapi/scripts/subscription/ubble/update_pending_ubble_applications.py

```diff
@@ -0,0 +1,34 @@
+"""
+Sometimes the ubble webhook fails. This is a problem because the application is still pending.
+we want to be able to retrieve the processed applications even if Ubble failed, hence this script
```

Comment:
pourquoi "even if ubble failed" ? on n'en sait rien je pense.

-> we want to be able to update their status.